docs/internals/sync.md

```diff
@@ -143,23 +142,17 @@ A checkpoint identifier is attached to the following message types:
 - `command=request_sync_checkpoint`: Requested checkpoint identifier.
 - `command=sync_checkpoint`: Current checkpoint identifier of sender.
 
-### Canonical Checkpoint
-
-A _canonical_ checkpoint is a checkpoint:
-1. with an op committed atop it by the primary (discovery via `command=commit`), or
-2. that a majority quorum of replicas have reached (discovery via `command=ping`), or
-
 ### Sync Target
 
-A _sync target_ is the [checkpoint identifier](#checkpoint-identifier) of the checkpoint that the superblock-syn is syncing towards.
+A _sync target_ is the [checkpoint identifier](#checkpoint-identifier) of the checkpoint that the superblock-sync is syncing towards.
 
 Not all checkpoint identifiers are valid sync targets.
 
-- Every sync target **must** be a [canonical checkpoint](#canonical-checkpoint).
-  (TODO: Once prepares include the checkpoint identifier, this requirement can be removed.)
-- Every sync target op **must** either:
-  - be greater-than-or-equal-to the replica's current checkpoint op.
-  - be equal to the replica's current checkpoint op, but with a different target id (if our checkpoint diverged).
+Every sync target **must**:
+- have an op greater than the replica's current checkpoint op.
+- either:
+  - be committed atop (ensuring that it has been reached by a quorum of replicas), or
```

Comment:
Could you expand what "committed atop" means? To disambiguate between `op_checkpoint + 1` and
`Checkpoint.trigger_for(op_checkpoint) + 1`, which are both valid interpretations for the current
wording.